### ISSUE TITLE
Add Events Content

### ISSUE THREAD

**Issue by jnschrag** [CSIS-iLab/health-commission#34]
These are the bulk of the content and should live in the `_events` directory.
- [ ] Add events to `frasco.config.js` under `jekyll`

Front Matter:
- [ ] Title
- [ ] Date
- [ ] Excerpt
- [ ] Content Type (set to event by default, make this field editable by developers only)
- [ ] Tags (list of keywords)
- [ ] Image
- [ ] Image Caption
- [ ] Image Credit
- [ ] Documents (this field should allow the program to upload PDFs with nice titles)

**Comment by aleesteele** [comment]
Q: Would you prefer a text field with a default `event` or a dropdown select field, with data files as the source for the list?

**Comment by jnschrag** [comment]
Let's make this is a text field with a default of 'event'. Be sure to check the option that makes this editable by developers only.

**Comment by aleesteele** [comment]
Ok-- And confirming: tags/keywords are by default, sortable lists?

**Comment by jnschrag** [comment]
> Ok-- And confirming: tags/keywords are by default, sortable lists?

Yes. We don't use Forestry's 'tag' field because it doesn't allow us to autocomplete/pull from the `keywords` collection.
